src/spatial_ref/srs.rs

```diff
@@ -439,12 +439,18 @@ impl SpatialRef {
     }
 
     #[cfg(major_ge_3)]
+    #[deprecated note = "use `axis_mapping_strategy` instead"]
     pub fn get_axis_mapping_strategy(&self) -> gdal_sys::OSRAxisMappingStrategy::Type {
```

Comment:
This method was released with 0.7.0, so I gracefully deprecated it. The other methods changed haven't been released yet so I just changed them inline.